### PR TITLE
Adds runtime-stats option to CLI

### PR DESCRIPTION
**Pull request by efrancisworks**
## Description
Hides runtime stats from Presto CLI in debug mode. Instead use --runtime-stats to show query statistics.

## Motivation and Context
Resolves visibility issues explained in #22041 .

## Impact
User must opt in to view runtime stats. 

## Test Plan
Use the --debug option and make a query. Then use the --runtime-stats option to view difference.

## Contributor checklist

- [x] Please make sure your submission complies with our [development](https://github.com/prestodb/presto/wiki/Presto-Development-Guidelines#development), [formatting](https://github.com/prestodb/presto/wiki/Presto-Development-Guidelines#formatting), [commit message](https://github.com/prestodb/presto/wiki/Review-and-Commit-guidelines#commit-formatting-and-pull-requests), and [attribution guidelines](https://github.com/prestodb/presto/wiki/Review-and-Commit-guidelines#attribution).
- [x] PR description addresses the issue accurately and concisely.  If the change is non-trivial, a GitHub Issue is referenced.
- [x] Documented new properties (with its default value), SQL syntax, functions, or other functionality.
- [x] If release notes are required, they follow the [release notes guidelines](https://github.com/prestodb/presto/wiki/Release-Notes-Guidelines).
- [ ] Adequate tests were added if applicable.
- [ ] CI passed.

## Release Notes
Please follow [release notes guidelines](https://github.com/prestodb/presto/wiki/Release-Notes-Guidelines) and fill in the release notes below.

```
== NO RELEASE NOTE ==
```

### CODE REVIEW

**Comment by tdcmeehan** [comment]
@efrancisworks looks good, but could you please rebase?  This will fix a flaky test that is currently blocking this PR.

**Comment by tdcmeehan** [comment]
@efrancisworks one last thing, please fix the commit message, suggested: `Add --runtime-stats option to CLI`

File: presto-cli/src/main/java/com/facebook/presto/cli/StatusPrinter.java [hunk 1]
```diff
@@ -203,7 +205,7 @@ public void printFinalInfo(Optional<Long> clientStopTimestamp)
                 stats.getProgressPercentage().orElse(0.0));
         out.println(splitsSummary);
 
-        if (debug) {
+        if (runtime) {
```
Comment (tdcmeehan) [inline presto-cli/src/main/java/com/facebook/presto/cli/StatusPrinter.java:208]: I remember an earlier version which is different--this seems to swap out the runtime for debug flags entirely.  I think you want to leave this, and add a nested check on line 243.

File: presto-cli/src/main/java/com/facebook/presto/cli/StatusPrinter.java [hunk 2]
```diff
@@ -223,7 +225,7 @@ public void printFinalInfo(Optional<Long> clientStopTimestamp)
             reprintLine(perNodeSummary);
 
             // Parallelism: 5.3
-            out.println(format("Parallelism: %.1f", parallelism));
+            out.printf("Parallelism: %.1f%n", parallelism);
```
Comment (tdcmeehan) [inline presto-cli/src/main/java/com/facebook/presto/cli/StatusPrinter.java:228]: Unrelated change?

**Review by tdcmeehan — CHANGES_REQUESTED** [review]

File: presto-cli/src/main/java/com/facebook/presto/cli/StatusPrinter.java (hunk 1 above)
Comment (efrancisworks) [reply presto-cli/src/main/java/com/facebook/presto/cli/StatusPrinter.java]: Got it.

File: presto-cli/src/main/java/com/facebook/presto/cli/StatusPrinter.java [hunk 3]
```diff
@@ -204,48 +206,50 @@ public void printFinalInfo(Optional<Long> clientStopTimestamp)
...
+
+                // Per Node: 3.5 parallelism, 83.3K rows/s, 0.7 MB/s
+                String perNodeSummary = format("Per Node: %.1f parallelism, %5s rows/s, %8s",
+                        parallelism / nodes,
+                        formatCountRate((double) stats.getProcessedRows() / nodes, serverSideWallTime, false),
+                        formatDataRate(bytes(stats.getProcessedBytes() / nodes), serverSideWallTime, true));
+                reprintLine(perNodeSummary);
+
+                // Parallelism: 5.3
+                out.printf("Parallelism: %.1f", parallelism);
```
Comment (tdcmeehan) [inline presto-cli/src/main/java/com/facebook/presto/cli/StatusPrinter.java:229]: Why the change form println to print without newline?

File: presto-cli/src/main/java/com/facebook/presto/cli/StatusPrinter.java (hunk 2 above)
Comment (efrancisworks) [reply presto-cli/src/main/java/com/facebook/presto/cli/StatusPrinter.java]: My IDE made this change and I forgot to undo it.

File: presto-cli/src/main/java/com/facebook/presto/cli/StatusPrinter.java (hunk 3 above)
Comment (efrancisworks) [reply presto-cli/src/main/java/com/facebook/presto/cli/StatusPrinter.java:229]: Oops, missed the format. I was returning it to the original version.

**Review by tdcmeehan — APPROVED** [review]

File: presto-cli/src/main/java/com/facebook/presto/cli/ClientOptions.java [hunk 4]
```diff
@@ -110,6 +110,9 @@ public class ClientOptions
     @Option(name = "--debug", title = "debug", description = "Enable debug information")
     public boolean debug;
 
+    @Option(name = "--runtime-stats", title = "runtime stats", description = "Enable run time stats information")
```
Comment (tdcmeehan) [inline presto-cli/src/main/java/com/facebook/presto/cli/ClientOptions.java:113]: One final nit, please mention that this flag must be enabled in conjunction with the --debug flag.

**Review by tdcmeehan — APPROVED** [review]
